Makefile

```diff
@@ -0,0 +1,8 @@
+
+build:
+	mvn package -DskipTests=true
```

Comment:
are there any tests?  If so, a recipe should be made for that

Reply:
Added the recipe for tests.